src/Plugin/Field/ReactParagraphsFields/Ckeditor.php

```diff
@@ -62,4 +69,17 @@ public function getFieldInfo(array $field_element, FieldConfigInterface $field_c
     return $info;
   }
 
+  /**
+   * Get all filter formats allowed for the current user.
+   *
+   * @return \Drupal\filter\FilterFormatInterface[]
+   *   Keyed array of filter formats.
+   *
+   * @codeCoverageIgnore
```

Comment:
Not a big deal but aren't you supposed to supply a mock global function in your test instead of ignoring it? This is just for coverage so I'm not too worried just looking to learn.

Reply:
its an option. but its so much work. its easier to mock what is returned from the method.